app/(afterLogin)/(timesheetInformation)/timesheet/my-timesheet/_components/leaveBalance/index.tsx

```
@@ -66,9 +65,9 @@ const LeaveBalance = () => {
                 setSwiper(swiper);
               }}
             >
-              {data.items
-                .filter((item) => item.leaveType)
-                .map((item) => (
+              {data.items?.items
+                .filter((item:any) => item.leaveType)
```

ESLint

[error] Insert `·`

(prettier/prettier)